css/style.css

```diff
@@ -0,0 +1,252 @@
+body {  
```

Comment:
Work on indentation, removing dead code, and vertical white space in this file to make your code more readable and consistent!